cuppa/src/main/java/com/hartwig/hmftools/cup/cli/ShellCommand.java

```diff
@@ -68,77 +68,57 @@ public ShellCommand timeout(long timeout)
         return this;
     }
 
-    public String toString()
-    {
-        return mProcessBuilder.command().toString();
-    };
-
     public ShellCommand run()
     {
         checkComplete();
-
         try
         {
             if(mShowCommand)
-                CUP_LOGGER.info("Running command: " + this);
+                CUP_LOGGER.info("Running command: [{}]", this);
 
             Process process = mProcessBuilder
                     .redirectErrorStream(true) // Merge stdout and stderr
                     .start();
 
-            mStdout = captureStdout(process, mLogLevel);
-
             mComplete = process.waitFor(mTimeout, mTimeoutUnit);
-            process.destroy();
-
+            CUP_LOGGER.info("Command [{}] complete", this);
```

Comment:
Change to `CUP_LOGGER.info("Command complete: [{}]", this);`. Otherwise 'complete' is buried at the end of the log message